src/enact/invocations.py

```diff
@@ -775,7 +781,8 @@ def invoke(
       except raise_on_errors:
         raise
       except Exception:  # pylint: disable=broad-exception-caught
-        if not builder.completed:
+        wrap_error = builder.exception_wrapped and not wrap_exceptions
+        if not builder.completed or wrap_error:
```

Comment:
maybe rename `wrap_error` to `should_raise_wrapped_error`

Reply:
Agreed, done.